lock/client/client.go

```diff
@@ -0,0 +1,136 @@
+// Package client implements a client to the volmaster to acquire locks.
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/contiv/volplugin/config"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// Driver is the main force behind the lock module, it controls all methods and
+// several variables.
+type Driver struct {
+	master         string
+	mountStopChans map[string]chan struct{}
```

Comment:
I might have missed this, but I am not very clear on advantage of using a `struct{}` vs say `interface{}` or `bool` here. We don't seem to be interpreting/using the `struct` received on this channel anywhere, right?